src/schema/auth.ts

```diff
@@ -0,0 +1,31 @@
+type AccessType = 'online' | 'offline';
+type Provider = 'google' | 'microsoft';
```

Comment:
is yahoo missing here?

Reply:
IDK if we support Yahoo for other stuff I know passthru does not